apps/omg/test/support/test_helper.ex

```diff
@@ -158,6 +158,23 @@ defmodule OMG.TestHelper do
     {:ok, full_path, file}
   end
 
+  def call_while(caller, matcher, time \\ 5_000, step \\ 10) do
```

Comment:
this is actually a very unreadable implementation and I think we want to revert to the previous one.